### PR TITLE
Add mergeable prow external plugin

### PR DESCRIPTION
**Pull request by mkumatag**
Fixes: #313

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -270,3 +270,18 @@ external_plugins:
...
+  ocp-power-automation/ocp4-upi-powervm:
+  - name: mergeable
+    events:
+      - issue_comment
+      - pull_request
+  ocp-power-automation/ocp4-playbooks:
+  - name: mergeable
+    events:
+      - issue_comment
+      - pull_request
```
[the remark this answers is not in the document]
Comment (mkumatag) [reply config/prow/plugins.yaml:287]: I don't mind enabling this to entire repo, but our hook is not configured for all the repos in the org, not sure how that will behaves, may be will try to restrict to only to the repos which got +2 restrictions.

[the remark this answers is not in the document]
Comment (mkumatag) [reply config/prow/plugins.yaml:287]: added an entry for the `ocp-power-automation/openshift-install-power` repo

**Review by yussufsh — APPROVED** [review]
/lgtm

**Pull request merged**